packages/sdk/src/oracle/client.ts

```diff
@@ -162,6 +164,19 @@ export class Client {
   }
 }
 
+function makeProvider(rpcUrls: string[]): FallbackProvider {
```

Comment:
this logic was pulled out of the write class unchanged for the most part. just so I can use the provider during configuration